### PR TITLE
feat(classnames): updated classnames import for a consistent import name

### LINKED ISSUE #799: [classnames] unify import name
We're not using a consistent name for usages of the `classnames` package. 

`import cn from 'classnames';`
`import cx from 'classnames';`
`import classnames from 'classnames';`
`import classNames from 'classnames';`

We need to decide on one and refactor all usages to fall in line.

### PR DESCRIPTION
**Pull request by soniadeveloper**
Closes #799

**Summary**

The classnames import used multiple names depending on the file (classnames, classNames, cx, cn), so the import names were updated so only one name is used (classnames).

**Change List (commits, features, bugs, etc)**

- Updated classnames import for a consistent import name
- Opened issue #1325 to track the progress for a potential ESLint rule that will maintain consistent naming for future imports.

**Acceptance Test (how to verify the PR)**

- In VSCode, do a project wide search for "classnames"
- This search should show that each time classnames is imported, it will be imported as "classnames"
Screen Shot 2020-06-18 at 10 20 27 AM

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -76,6 +76,23 @@
...
+        "name": "classnames",
+        "importNames": ["cx"],
+        "message": "Please use classnames from classnames instead."
+      },
+      {
+        "name": "classnames",
+        "importNames": ["cn"],
+        "message": "Please use classnames from classnames instead."
+      }]
+    }],
```
Comment (tay1orjones) [inline .eslintrc:95]: Does this close #1325 as well? I think we can change the level to `error`
```suggestion
    "no-restricted-imports": ["error", {
      "paths": [{
        "name": "classnames",
        "importNames": ["classNames"],
        "message": "Please import classnames as 'classnames'."
      },
      {
        "name": "classnames",
        "importNames": ["cx"],
        "message": "Please import classnames as 'classnames'."
      },
      {
        "name": "classnames",
        "importNames": ["cn"],
        "message": "Please import classnames as 'classnames'."
      }]
    }],
```

Comment (soniadeveloper) [reply .eslintrc:95]: @tay1orjones That rule doesn't work because it doesn't trigger any error messages when the importNames property is used. :(

Comment (tay1orjones) [reply .eslintrc]: 👍  cool. lets contain the eslint exploration in a separate PR tied to #1325

**Pull request merged**